eq-author/src/components/RichTextEditor/index.js

```diff
@@ -200,7 +200,10 @@ class RichTextEditor extends React.Component {
       this.focus();
     }
     /*eslint-disable react/no-did-update-set-state */
-    if (prevProps.value !== this.props.value) {
+    if (
+      prevProps.value !== this.props.value &&
+      typeof this.props.value === "string"
```

Comment:
Whats this type check for out of intrest, can it ever be anything other than a string?

Reply:
At the beginning for some fields it is `null`